network/syncing/syncer.go

```diff
@@ -92,11 +96,32 @@ func (s *syncer) SyncHighestDecided(
 			logger.Debug("could not encode signed message", zap.Error(err))
 			return false
 		}
-		handler(spectypes.SSVMessage{
+
+		ssvMessage := &spectypes.SSVMessage{
 			MsgType: spectypes.SSVConsensusMsgType,
 			MsgID:   id,
 			Data:    raw,
-		})
+		}
+
+		// TODO: consider uncommenting
+		if s.msgValidator != nil {
```

Comment:
when is `msgValidator` nil?

Reply:
It seems like now it's never nil, removing this check